### ISSUE TITLE
htfpp cannot handle non-ASCII characters in source files

### ISSUE THREAD

**Issue by koterpillar** [skogsbaer/HTF#47]
If a source file contains non-ASCII characters, even in comments, then any tests that come _after_ those characters are not picked up into `htf_thisModulesTests`.

Example file:
```haskell
{-# OPTIONS_GHC -F -pgmF htfpp #-}

module Main where

import Test.Framework

test_first = do
    -- 世界不是英文的
    assertEqual True True

test_second = do
    assertEqual True False

main = htfMain htf_thisModulesTests
```

Output when run:
```
[TEST] Main:first (htf-unicode-bug.hs:7)
+++ OK (0ms)

* Tests:     1
* Passed:    1
* Pending:   0
* Failures:  0
* Errors:    0
* Timed out: 0
* Filtered:  0

Total execution time: 0ms
```

Output of `htfpp` on the file:
```haskell
#line 1 "<stdin>"
{-# OPTIONS_GHC -F -pgmF htfpp #-}

module Main where

import Test.Framework

test_first = do
    -- 世界不是英文的
    (assertEqual_ (makeLoc "<stdin>" 9)) True True

test_second = do
    (assertEqual_ (makeLoc "<stdin>" 12)) True False

main = htfMain htf_Main_thisModulesTests


htf_Main_thisModulesTests :: TestSuite
htf_Main_thisModulesTests = makeTestSuite "Main" [
{-# LINE 7 "<stdin>" #-}
    makeUnitTest "first" (makeLoc "<stdin>" 7) test_first
  ]

```

Note how `test_second` is missing from `htf_Main_thisModulesTests`.

**Comment by skogsbaer** [comment]
Thanks for your bugreport! The bug will be fixed in the upcoming release, where we use the lexer from cpphs.

**Comment by skogsbaer** [comment]
should be fixed in the newest version on hackage (0.13.0.0). Please reopen if the problem persists.
